listenbrainz/webserver/views/art_api.py

```diff
@@ -254,6 +254,10 @@ def _cover_art_yim_albums(user_name, stats):
     cac = CoverArtGenerator(current_app.config["MB_DATABASE_URI"], 3, 250)
     image_urls = []
     selected_urls = set()
+
+    if stats.get("top_releases") is None:
```

Comment:
I'm not too worried either way but this could also have been `if not stats.get("x")` (unless there are falsy items?), or even `if "x" not in stats` (which I'm a bit more of a fan of, but not enough to force you to change it)

Reply:
Makes sense but `"x" not in stats` doesn't handle the case of null items in dict.